### DISCUSSION TITLE
Simple Tables

### DISCUSSION THREAD

**Discussion by tbohaning** [jendrikseipp/rednotebook#702] [Ideas]
Hi,

Was wondering if it would be possible to support simple table functionality.

A lot of what I'm using rednotebook for contains data that is easier to read in a table.

The simplest solution that I can think of would look like the following.

The marker would be to use a "|" to separate the fields and each line would be a row in the table. The table would need 2 blank lines before and after similar to the way that lists are handled now.. The default would be a left justified with wrapping automatically turned on. 

I'm sure there are a lot of features/functionality that I'm leaving out like fixed column widths, headers, footers, colors, etc. But the goal really would be something simple in keeping with the general functionality of the tool.

Thoughts?

**Comment by jendrikseipp** [comment]
The underlying markup tool "txt2tags" supports tables: https://txt2tags.org/markup.html

However, since we are considering switching to Markdown, it is unlikely that the txt2tags table format will be supported in the future.
